README.md

```diff
@@ -95,12 +95,24 @@ containerized sandbox environment.
 
 ## Supported Protocol Versions
 
-**ZDM Proxy supports protocol versions v3, v4, DSE_V1 and DSE_V2.**
+**ZDM Proxy supports protocol versions v2, v3, v4, DSE_V1 and DSE_V2.**
 
 It technically doesn't support v5, but handles protocol negotiation so that the client application properly downgrades
 the protocol version to v4 if v5 is requested. This means that any client application using a recent driver that supports
 protocol version v5 can be migrated using the ZDM Proxy (as long as it does not use v5-specific functionality).
 
+ZDM Proxy requires origin and target clusters to have at least one protocol version in common. It is therefore not feasible
+to configure Apache Cassandra 2.0 as origin and 3.x / 4.x as target. Below table displays protocol versions supported by
+various C* versions:
+
+| Apache Cassandra | Protocol Version |
+|------------------|------------------|
+| 2.0              | V2               |
+| 2.1              | V2, V3           |
+| 2.2              | V2, V3, V4       |
+| 3.x              | V3, V4           |
+| 4.x              | V3, V4, V5       |
+
```

Comment:
With [OSS C* `5.0` around the corner](https://lists.apache.org/thread/1jtjt0h506y0kc4jt2v73jlgbm67mcb5), should we plan for it too (or) is it too ambitious?

Reply:
V5 is definitely work for another PR. Moreover, as long as Astra and C* 5.x / DSE supports V4, they can still use ZDM for migration. Most CQL drivers do not support protocol V5 yet.